src/qt/optionsmodel.cpp

```diff
@@ -461,6 +471,32 @@ void OptionsModel::setStakeSplitThreshold(const CAmount nStakeSplitThreshold)
     }
 }
 
+/* Update Custom Fee value in wallet */
+void OptionsModel::setUseCustomFee(bool fUse)
```

Comment:
Instead of use a boolean, could set the custom fee to a negative number and use the same field for both action.